Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Basic docker image for Monocle
+# Usage:
+#   docker build -t Monocle
+#   docker run -d --name Monocle -P Monocle
+
+FROM python:3.6
```

Comment:
I think the accounts.pickle could be a thing, too
as if you don't have your phone IDs in the accounts.csv they would be generated everytime